### ISSUE TITLE
Dishonored 2 Version Possible?

### ISSUE THREAD

**Issue by RisingEclipse** [adm244/Dishonored-FullRegen#1]
Played your mod a few runs on the original dishonored, and I'm looking forward to it for the Dishonored 2 game. Is there any chance of it being ported to it?

**Comment by adm244** [comment]
Yes, it's possible. I even looked at it some time ago, but then got distracted by other things. If I ever get back to it, that will be after I finish these other projects.

**Comment by RisingEclipse** [comment]
sorry for the late response! I would love this. Your mod made many of my playthroughs more enjoyable since I didn't have to concern myself with mana conservation while also keeping some balance to it. 

I hope you get the time to eventually work on porting this mod to Dishonored 2.
